### PR TITLE
[editor] Descriptions for PromptSchema (OpenAIChatModelParser)

### PR DESCRIPTION
**Pull request by rholinshead**
# [editor] Descriptions for PromptSchema (OpenAIChatModelParser)

The settings renderers already support info tooltips for 'description' specified in the schema, so just add the descriptions as an example:

Screenshot 2024-01-04 at 5 10 07 PM

### CODE REVIEW

File: python/src/aiconfig/editor/client/src/components/SettingsPropertyRenderer.tsx [hunk 1]
```diff
@@ -41,7 +41,7 @@ export function PropertyLabel(props: {
     <Group spacing="xs">
       <Text size="md">{propertyName}</Text>
 
-      <Tooltip label={propertyDescription} multiline>
+      <Tooltip label={propertyDescription} multiline maw={400}>
```
Comment (rossdanlm) [inline python/src/aiconfig/editor/client/src/components/SettingsPropertyRenderer.tsx:44]: What is `maw`?

Comment (rossdanlm) [reply python/src/aiconfig/editor/client/src/components/SettingsPropertyRenderer.tsx:44]: Alright big player I see you: https://github.com/lastmile-ai/aiconfig/blob/3dcf70bcddda7f7041c3bfb6e5d4b6c9a7abd4ab/python/src/aiconfig/editor/client/src/components/SettingsPropertyRenderer.tsx#L64

**Review by rossdanlm — APPROVED** [review]
It's beautiful

File: python/src/aiconfig/editor/client/src/components/SettingsPropertyRenderer.tsx (hunk 1 above)
Comment (rholinshead) [reply python/src/aiconfig/editor/client/src/components/SettingsPropertyRenderer.tsx:44]: It's `maxWidth` style setting for most mantine components. They expose some basic style modifiers (margin, width, etc.) on most components so you can modify their styles inline without needing classes/css.

**Comment by rholinshead** [comment]
Ok, weird Github UI state here. I hit 'Merge pull request' and it was stuck spinning for a minute so I reloaded the page and now the button is disabled. It actually did get merged but the PR is still open for some reason. Closing to prevent more weirdness

**Pull request closed without merging**